tests/consensus_test.rs

```diff
@@ -114,8 +125,9 @@ impl NodeForTest {
 
 #[tokio::test]
 async fn test_basic_consensus() {
+    let env_filter = EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("info"));
```

Comment:
Let's keep this to `warn`. We could disable entirely for tests, but I've found this useful for debugging.